app/_src/mesh/features/openshift-quickstart.md

```diff
@@ -0,0 +1,184 @@
+    ```
+
+1. Open a second terminal and port-forward to reach the mesh GUI:
+    ```bash
+    kubectl port-forward svc/kong-mesh-control-plane -n kong-mesh-system 5681:5681
+    ```
+
+    You should be able to reach the Kong-Mesh UI at `http://localhost:5681/gui`.
+
+
+1. Last, do some prep work for the sidecar itself so sidecars will startup successfully. Apply the kong-mesh-sidecar scc and corresponding container patches:
+    ```bash
+    kubectl create -f kong-mesh/kong-mesh-sidecar-scc.yaml
+    kubectl apply -f kong-mesh/container-patch.yaml 
+    ```
+
+## Deploy KIC on Mesh
+
+1. First we're going to deploy KIC as our ingress controller:
+    ```bash
+    kubectl create namespace kong 
+    kubectl label namespace kong kuma.io/sidecar-injection=enabled
+    oc adm policy add-scc-to-user kong-mesh-sidecar system:serviceaccount:kong:kong-kong
+    ```
+
+1. Grab the latest chart:
+    ```bash
+    helm repo add kong https://charts.konghq.com
+    helm repo update
+    ```
+
+1. Then run the install:
+    ```bash
+    helm install kong kong/kong -n kong \
+    --set ingressController.installCRDs=false \
+    --set podAnnotations."kuma\.io/mesh"=default \
+    --set podAnnotations."kuma\.io/gateway"=enabled
+    export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].hostname}" service -n kong kong-kong-proxy)
+    curl -i $PROXY_IP
+    ```
+
+    The expected output is:
+    ```json
+    HTTP/1.1 404 Not Found
+    Date: Tue, 02 May 2023 15:40:05 GMT
+    Content-Type: application/json; charset=utf-8
+    Connection: keep-alive
+    Content-Length: 48
+    X-Kong-Response-Latency: 0
+    Server: kong/3.2.2
+
+    {"message":"no Route matched with those values"}%
+    ```
+
+## Deploy Bookinfo
+
+1. Allow all the service accounts to use the kong-mesh-sidecar scc:
+    ```bash
+    oc adm policy add-scc-to-user kong-mesh-sidecar system:serviceaccount:bookinfo:bookinfo-details
+    oc adm policy add-scc-to-user kong-mesh-sidecar system:serviceaccount:bookinfo:bookinfo-productpage
+    oc adm policy add-scc-to-user kong-mesh-sidecar system:serviceaccount:bookinfo:bookinfo-ratings
+    oc adm policy add-scc-to-user kong-mesh-sidecar system:serviceaccount:bookinfo:bookinfo-reviews
+    ```
+
+1. Next, deploy bookinfo. This bookinfo app has been paired down to work with Kong Mesh in evaluation mode. We only have 5 sidecars we can deploy on evaluation mode is all.
+    ```bash
+    kubectl create namespace bookinfo
+    kubectl label namespace bookinfo kuma.io/sidecar-injection=enabled
+    kubectl apply -f bookinfo/bookinfo.yaml -n bookinfo
+    ```
+
+1. Last we want to create an ingress resource to expose bookinfo to the outside world.
+    ```bash
+    kubectl apply -f bookinfo/ingress-productpage.yaml -n bookinfo
+    ```
+
+1. Now! Validate it's all up and running. Navigate to your browswer to `http://$PROXY_IP/productpage` and you should be able to see productpage with a majority of the bells and whistles.
```

Comment:
Is the purpose of using bookinfo here to see this page? Or are there other things customers can do with bookinfo now that it's hooked up to Mesh? I just want to make the connection between what customers would do here with bookinfo and what they would do in their own production environments.

Reply:
They can test out more Kong Mesh features. I stopped here for the validation because this stopping point confirmed that all pods and sidecars are healthy.